packages/plugin-essentials/sources/commands/set/version/sources.ts

```diff
@@ -33,7 +34,7 @@ const updateWorkflow = ({branch}: {branch: string}) => [
 
 const buildWorkflow = ({plugins, noMinify}: {noMinify: boolean, plugins: Array<string>}, output: PortablePath, target: PortablePath) => [
   [`yarn`, `build:cli`, ...new Array<string>().concat(...plugins.map(plugin => [`--plugin`, ppath.resolve(target, plugin as Filename)])), ...noMinify ? [`--no-minify`] : [], `|`],
-  [`mv`, `packages/yarnpkg-cli/bundles/yarn.js`, npath.fromPortablePath(output), `|`],
+  [IS_WIN32 ? `move` : `mv`, `packages/yarnpkg-cli/bundles/yarn.js`, npath.fromPortablePath(output), `|`],
```

Comment:
Ideally, it would be worth making some kind of mechanism for platform-specific commands, but this should work